statler_api/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
-
-# Reference the views contained in this app. Will the API actually use views?
+# Reference the views contained in this app.
 from . import views
 
 # This is where the magical regex mapping happens.
 urlpatterns = [
-    url(r'^health-check$', views.healthCheck, name='healthCheck'),
```

Comment:
I think it may be valuable to keep a health check endpoint in place for debugging (to make sure a new machine is properly configured, for example). It may be good to modify the name to match the 'play-list' convention below.